Cargo.toml

```diff
@@ -8,7 +8,7 @@ authors = ["Avail Team"]
 
 [dependencies]
 kate-recovery = { version = "0.8", git="https://github.com/availproject/avail-core", tag = "da-primitives/v0.4.6" }
-avail-subxt = { git = "https://github.com/availproject/avail.git", tag = "v1.6.0" }
```

Comment:
Does it make sense to depend only on tagged versions of avail, so we are confident in which node version should work? That means we have to enforce tagging when changes are on both LC and the node, but it seems as good trade off.

Reply:
Yes, but there's one commit I'm dependent on, which is just after the tag and I don't know what's the proper way to handle this. Tag 1.6.1? Or something else?

Reply:
Actually if that is unstable version on develop branch which didn't landed into the main, we cannot do something meaningful. But in that case LC is also in unstable state. Soooo :man_shrugging: